dev/tasks/r/github.packages.yml

```diff
@@ -82,14 +82,24 @@ jobs:
       {{ macros.github_checkout_arrow()|indent }}
       {{ macros.github_change_r_pkg_version(is_fork, '${{ needs.source.outputs.pkg_version }}')|indent }}
       {{ macros.github_install_archery()|indent }}
+
       - name: Build libarrow
         shell: bash
         env:
-          UBUNTU: {{ '${{ matrix.config.version}}' }}
+          SCCACHE_BUCKET: {{ '${{ secrets.SCCACHE_BUCKET }}' }}
+          SCCACHE_ID: {{ '${{ secrets.AWS_ACCESS_KEY_ID }}' }}
+          SCCACHE_KEY: {{' ${{ secrets.AWS_SECRET_ACCESS_KEY }}' }}
+          SCCACHE_PREFIX: sccache

```

Comment:
SCCACHE_KEY_PREFIX perhaps?

Reply:
I am happier with this name `SCCACHE_S3_KEY_PREFIX` too as it gives me a little more context.